utplsql-maven-plugin/src/test/resources/type-mapping-project/pom.xml

```diff
@@ -0,0 +1,94 @@
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.utplsql</groupId>
+    <artifactId>utplsql-maven-plugin-test</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>maven-plugin</packaging>
+
+    <name>utplsql-maven-plugin Maven Plugin Test</name>
+    
+    
+     <properties>
+      	<dbUrl>jdbc:oracle:thin:@127.0.0.1:1521:xe</dbUrl>  
+		<!-- 
+ 		<dbUser>ut3</dbUser> 
+  		<dbPass>XNtxj8eEgA6X6b6f</dbPass> 
+  		-->
+    </properties>
+    
+    <build>
+    
+        <plugins>
+            <plugin>
+                <groupId>${pom.groupId}</groupId>
+                <artifactId>utplsql-maven-plugin</artifactId>
+                <version>${pom.version}</version>
+                <goals>
+                    <goal>test</goal>
+                </goals>
+                <configuration>
+                	<!--  Mandatory Attributes -->
+                	
+					<ignoreFailure>false</ignoreFailure>
+					
+					<paths>
+            			<path>app</path>
+          			</paths>
+					
+					<reporters>
+						<reporter>
+								<name>UT_COVERAGE_SONAR_REPORTER</name>
+								<fileOutput>utplsql/coverage-sonar-reporter.xml</fileOutput>
+								<consoleOutput>true</consoleOutput>
+						</reporter>
+						<reporter>
+								<name>UT_SONAR_TEST_REPORTER</name>
+								<fileOutput>utplsql/sonar-test-reporter.xml</fileOutput>
+								<consoleOutput>true</consoleOutput>
+						</reporter>
+					</reporters>
+					
+                    <sources>
+                    	<source>
+                    		<directory>src/test/resources/type-mapping-project/scripts/sources</directory>
+							<includes>
+								<include>**/*sql</include>
+							</includes>
+                    	</source>
+                    </sources>
+                    <sourcesRegexExpression>.*(\\|\/)(\w+)(\\|\/)(\w+)(\\|\/)(\w+).(\w{3})</sourcesRegexExpression>
+                    <sourcesOwnerSubexpression>2</sourcesOwnerSubexpression>
+                    <sourcesNameSubexpression>6</sourcesNameSubexpression>
+                    <sourcesTypeSubexpression>4</sourcesTypeSubexpression>
+                    <sourcesCustomTypeMapping>
+                    	<customTypeMapping>
+                    		<type>package body</type>
+                    		<customMapping>package_bodies</customMapping>
+                    	</customTypeMapping>
+                    </sourcesCustomTypeMapping>
+                    
+                    <tests>
+                    	<test>
+                    		<directory>src/test/resources/type-mapping-project/scripts/test</directory>
+							<includes>
+								<include>**/*sql</include>
+							</includes>
+                    	</test>
+                    </tests>
+                    <testsRegexExpression>.*(\\|\/)(\w+)(\\|\/)(\w+)(\\|\/)(\w+).(\w{3})</testsRegexExpression>
```

Comment:
Wirh utPLSQL v3.1 the regex can be of a simplified form. See doc.

Reply:
Could you point the documentation about this ? :)
I need to use the "owner" parameter ?